src/lightning/data/streaming/reader.py

```diff
@@ -294,16 +291,16 @@ def _get_folder_size(path: str) -> int:
     return size
 
 
-def _get_from_queue(queue: multiprocessing.Queue) -> Optional[Any]:
+def _get_from_queue(queue: multiprocessing.Queue, timeout: float = _DEFAULT_TIMEOUT) -> Optional[Any]:
     try:
         # Note: The timeout here should not be too short. We need to prevent the caller from aggressively
-        #   querying the queue and consuming too many CPU cycles.
```

Comment:
The comment applies to where this timeout is defined, so now that is has moved, you need to move the comment too :)

Reply:
This is always defined and there is also a timeout added above.